network/debinterfaces/parser_test.go

```diff
@@ -0,0 +1,727 @@
+// Copyright 2017 Canonical Ltd.
+// Licensed under the AGPLv3, see LICENCE file for details.
+
+package debinterfaces_test
```

Comment:
My eyes are starting to glaze over, and I feel like it is important to review this well. So I'm going to submit the first part of feedback for now, and then pick the rest of this up tomorrow.